### DISCUSSION TITLE
Saving Sessions started again - my automation from last year isn't working - is it me or has API/Integration changed?

### DISCUSSION THREAD

**Discussion by steevjo** [BottlecapDave/HomeAssistant-OctopusEnergy#1116] [Q&A]
Great integration - it successfully automated saving events throughout last winter for me - joined the event, and provided event times I could use to control our battery.

First saving session here today but it wasn't picked up by my auto-join automation (same one that worked last year). There's plenty of events detected but nothing triggers the automation
![save session](https://github.com/user-attachments/assets/ddab2216-2398-41f1-8cc4-29d0b1129f91)

Updated to latest 13.2.1 version - no change 

Any thoughts? Many thanks

**Comment by BottlecapDave** [comment]
If you look at the attributes of the event sensor, the upcoming session doesn't seem to be present. I'm currently away from my computer so can't check the raw data (as the integration filters out "test" sessions)

**Reply by BottlecapDave** [reply]
Please follow [this issue](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/1115) for more updates

**Reply by steevjo** [reply]
Will do - thanks

**Comment by deepanshu-sh** [comment]
same, came in to check if something has changed
